### DISCUSSION TITLE
Is there a setting to disable opening on the last viewed page?

### DISCUSSION THREAD

**Discussion by nhansendev** [xournalpp/xournalpp#5625] [Q&A] [answered]
When I open a xournal file it defaults to the last page I had open. This is not actually desirable to me, but I haven't found a preferences setting to turn it off. Did I miss something, or should this be a new feature request?

**Comment by bhennion** [comment, marked as answer]
No there isn't, but there is a command line option to open at a given page. Depending on your OS, you might use this as a workaround.
